common/djangoapps/microsite_configuration/backends/base.py

```diff
@@ -0,0 +1,160 @@
+"""
+Microsite configuration backend module.
+
+Contains the base class for microsite backends.
+
+"""
+
+from __future__ import absolute_import
+
+import abc
+import edxmako
+import os.path
+
+from django.conf import settings
+
+
+# pylint: disable=unused-argument
+class BaseMicrositeBackend(object):
```

Comment:
If this is an Abstract class it should probably be called. AbstractBaseMicrositeBackend